src/borg/crypto/key.py

```diff
@@ -216,9 +217,9 @@ def unpack_and_verify_manifest(self, data, force_tam_not_required=False):
             logger.warning('Manifest authentication DISABLED.')
             tam_required = False
         data = bytearray(data)
-        # Since we don't trust these bytes we use the slower Python unpacker,
-        # which is assumed to have a lower probability of security issues.
-        unpacked = msgpack.fallback.unpackb(data, object_hook=StableDict, unicode_errors='surrogateescape')
+        unpacker = remote.get_limited_unpacker('manifest')
```

Comment:
The rationale for that is a bit flawed since modifying the manifest data and modifying RPC is pretty similar in the attack model, but the slow packer isn't (and shouldn't) be used for RPC.